pkg/services/serviceaccounts/api/api.go

```diff
@@ -229,3 +231,56 @@ func (api *ServiceAccountsAPI) updateServiceAccount(c *models.ReqContext) respon
 
 	return response.JSON(http.StatusOK, resp)
 }
+
+// SearchOrgServiceAccountsWithPaging is an HTTP handler to search for org users with paging.
+// GET /api/org/users/search
+func (api *ServiceAccountsAPI) SearchOrgServiceAccountsWithPaging(c *models.ReqContext) response.Response {
+	ctx := c.Req.Context()
+	perPage := c.QueryInt("perpage")
+	if perPage <= 0 {
+		perPage = 1000
+	}
+	page := c.QueryInt("page")
+
+	if page < 1 {
+		page = 1
+	}
+
+	query := &models.SearchOrgUsersQuery{
+		OrgID:            c.OrgId,
+		Query:            c.Query("query"),
+		Page:             page,
+		Limit:            perPage,
+		User:             c.SignedInUser,
+		IsServiceAccount: true,
+	}
+
+	serviceAccounts, err := api.store.SearchOrgServiceAccounts(ctx, query)
+	if err != nil {
+		return response.Error(500, "Failed to get service accounts for current organization", err)
```

Comment:
```suggestion
		return response.Error(http.StatusInternalServerError, "Failed to get service accounts for current organization", err)
```

prefer http constants like the rest of the package

Reply:
yeas I prefer that as well 👍 nice